packages/cursorless-vscode/src/ide/vscode/hats/shapeAdjustments.ts

```diff
@@ -7,49 +7,52 @@ export interface HatAdjustments {
 
 export type IndividualHatAdjustmentMap = Record<HatShape, HatAdjustments>;
 
-export const DEFAULT_HAT_HEIGHT_EM = 0.29;
+export const DEFAULT_HAT_HEIGHT_EM = 0.38;
 export const DEFAULT_VERTICAL_OFFSET_EM = 0.032;
 
 export const defaultShapeAdjustments: IndividualHatAdjustmentMap = {
   default: {
-    sizeAdjustment: -14,
-    verticalOffset: -0.35,
+    sizeAdjustment: -15,
```

Comment:
Huh I thought the idea was to zero these all out. I guess easier to leave them full size and adjust here?

Reply:
Definitely. I could set zero for almost all of them since we're now actually taking the full viewport, but I still feel that actual "artistic" adjustments should be made in code and not in the svg themself. This is just so much simpler.

Reply:
> I still feel that actual "artistic" adjustments should be made in code and not in the svg themself.

Do/will people with custom hats have the ability to override the IndividualHatAdjustmentMap stuff?

Reply:
Yes the same adjustments are available in your vscode settings.

Reply:
Ah, I see it, thanks!

```
Cursorless:
Individual Hat Adjustments
Separate adjustments for each hat shape
Edit in settings.json
```

And once you click the edit link, you get a filled out dict with all zeros, exactly what I wanted!

![image](https://github.com/cursorless-dev/cursorless/assets/539362/7b433384-d4a8-436d-9e78-e15464fe08cf)
![image](https://github.com/cursorless-dev/cursorless/assets/539362/7d424b59-e2c2-4c78-9d85-a83fc1fb9acf)

Reply:
Seems fine to me. Prob easier to tweak this way while developing